service/core/client/src/main/resources/reference.conf

```diff
@@ -28,3 +28,11 @@ lagom.circuit-breaker {
   }
 }
 #//#circuit-breaker-default
+
+#//#web-socket-client-default
```

Comment:
These `#//#comment` markers are used to include snippets in the documentation.

For example, the `circuit-breaker-default` marker above is used in these two files:

- https://github.com/lagom/lagom/blob/master/docs/manual/java/guide/services/ServiceClients.md
- https://github.com/lagom/lagom/blob/master/docs/manual/scala/guide/services/ServiceClients.md

Would you be able to update those files to document this new configuration setting? You can use the snippet marker to include this fragment of the source code.

Reply:
Ok I see. I've added the correpsonding documentation with the reference to the config file.
However, I've done a little mess in the commits as I've committed&pushed my changes too soon. I'm not sure on the best way to clean that...

Reply:
@datalchemist, I'm not sure what you mean by 'mess', but I think the current state is correct. 

You wrapped the reference block you want to include in the docs and you referenced it in the docs.

The ordering of commits doesn't matter much to us. When we merge this PR, they will all get squashed into one single commit. Was that your concern?

Reply:
@renatocaval Ok, great then. Indeed, my concern was the multiple useless commits but if they are squashed when you merge, then that's fine.

Thanks for the feedback!